examples/forward/left_cerebellum_volume_source.py

```diff
@@ -46,10 +49,11 @@
 # %%
 # Plot the positions of each source space
 
-fig = mne.viz.plot_alignment(subject=subject, subjects_dir=subjects_dir,
+fig = mne.viz.plot_alignment(trans=trans, subject=subject,
+                             subjects_dir=subjects_dir,
                              surfaces='white', coord_frame='head',
                              src=src)
```

Comment:
Rather than pass+use an incorrect trans here, I'd rather do `coord_frame='mri'`

Reply:
After I had adjusted the roll and everything ;)